### PR TITLE
doc/cephadm: Adding doc for daemon_cache_timeout config option

### PR DESCRIPTION
**Pull request by rkachach**
Fixes: https://tracker.ceph.com/issues/58517

Signed-off-by: Redouane Kachach <rkachach@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [x] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/cephadm/services/index.rst
Comment (anthonyeleven) [inline doc/cephadm/services/index.rst:85]: Suggest double backticks \`\`ceph orch ps\`\`
"Daemon status is refreshed every 10 minutes by default."

File: doc/cephadm/services/index.rst
Comment (anthonyeleven) [inline doc/cephadm/services/index.rst:88]: backticks

**Review by anthonyeleven — CHANGES_REQUESTED** [review]

File: doc/cephadm/services/index.rst
Comment (anthonyeleven) [inline doc/cephadm/services/index.rst:88]: Suggest backticks around the second "ceph orch ps" on line 88.  I'm a markup junkie.

**Review by anthonyeleven — APPROVED** [review]
Suggest a minor formatting change, approving in advance so you don't have to wait for re-review.

File: doc/cephadm/services/index.rst
Comment (adk3798) [inline doc/cephadm/services/index.rst:88]:
```suggestion
   ``REFRESHED`` will indicate how old the information shown by ``ceph orch ps`` and similar commands is. Normally the information
```
grammar NIT

**Review by adk3798 — APPROVED** [review]
One grammar thing. Otherwise LGTM.

File: doc/cephadm/services/index.rst
Comment (rkachach) [reply doc/cephadm/services/index.rst]: Fixed

File: doc/cephadm/services/index.rst
Comment (rkachach) [reply doc/cephadm/services/index.rst]: fixed

File: doc/cephadm/services/index.rst
Comment (rkachach) [reply doc/cephadm/services/index.rst]: fixed

File: doc/cephadm/services/index.rst
Comment (rkachach) [reply doc/cephadm/services/index.rst]: fixed :+1:

File: doc/cephadm/services/index.rst
Comment (rkachach) [reply doc/cephadm/services/index.rst]: fixed and squashed :+1:

File: doc/cephadm/services/index.rst
Comment (adk3798) [inline doc/cephadm/services/index.rst:89]: Sorry, seem to have missed in initial review
```suggestion
   is refreshed every ``daemon_cache_timeout`` seconds unless the ``--refresh`` option is provided.
```
Also, do we want to add a little qualifier about what `--refresh` actually does. Something like "Note that while the option will tell cephadm to begin a refresh of the information, it does not mean the output received from the command will be up to date". Not sure if that's redundant given the first sentence you added. Wdyt?

Comment (rkachach) [reply doc/cephadm/services/index.rst]: > 

No prob, I rephrased the whole doc seeking for more clarity. Please, have a look and let me know if you have any comments/suggestions. Thanks.

**Review by adk3798 — APPROVED** [review]

**Pull request merged**
